test/language/expressions/arrow-function/arrow/apply_2.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using apply - 'this' is lexically bound
+es6id: 14.2
+---*/
+
+var b = {
+    method: () => this
+ }

```

Comment:
I see that for question 1 you have coverage below in the next test that verifies an encompassing object literal is not the value used for this.  That's good and sufficient to coverage imo.  This test here could focus on the four asserts using my sentinel suggestion above and not worry about an enclosing object literal.